_api-reference/profile.md

```diff
@@ -0,0 +1,448 @@
+              {
+                "name": "SimpleTopScoreDocCollector",
+                "reason": "search_top_hits",
+                "time_in_nanos": 77291
+              }
+            ]
+          }
+        ],
+        "aggregations": []
+      }
+    ]
+  }
+}
+```
+</details>
+
+## Response fields
+
+The response includes the following fields.
+
+Field | Data type | Description
+:--- | :--- | :---
+`profile` | Object | Contains profiling information.
+`profile.shards` | Array of objects | A search request can be executed against one or more shards in the index and a search may involve one or more indexes. Thus, the `profile.shards` array contains profiling information for each shard that was involved in the search.
+`profile.shards.id` | String | The shard ID of the shard in the `[node-ID][index-name][shard-ID]` format.
+`profile.shards.searches` | Array of objects | A search represents a query executed against the underlying Lucene index. Most search requests execute a single search against a Lucene index, but some search requests can execute more than one search. For example, including a global aggregation results in a secondary `match_all` query for the global context. The `profile.shards` array contains profiling information about each search execution.
+[`profile.shards.searches.query`](#the-query-object) | Array of objects | Profiling information about the query execution.
+`profile.shards.searches.rewrite_time` | Integer | All Lucene queries are rewritten. A query and its children may be rewritten more than once, until the query stops changing. The rewriting process involves performing optimizations, such as removing redundant clauses or replacing query path with a more efficient one. After the rewriting process, the original query may change significantly. The `rewrite_time` field contains the cumulative total rewrite time for the query and all its children, in nanoseconds.
+[`profile.shards.searches.collector`](#the-collector-object) | Array of objects | Profiling information about the Lucene collectors that ran the search.
+[`profile.shards.aggregations`](#aggregations) | Array of objects | Profiling information about the aggregation execution.
+
+### The `query` object
+
+The `query` object contains the following fields.
+
+Field | Data type | Description
+:--- | :--- | :---
+`type` | String | The Lucene query type into which the search query was rewritten. Corresponds to the Lucene class name (which often has the same name in OpenSearch).
+`description` | String | Contains Lucene explanation of the query. Helps differentiate queries with the same type.
+`time_in_nanos` | String | The time the query took to execute, in nanoseconds. In a parent query, the time is inclusive of execution times of all the child queries.
+[`breakdown`](#the-breakdown-object) | Object | Contains timing statistics about low-level Lucene execution.
+`children` | Array of objects | Contains information about all subqueries for this query.
```

Comment:
For better clarity, should we also mention that the `children` field is optional? If there is no subquery, then the `children` field won't be return.